librz/core/cmd/cmd_help.c

```diff
@@ -430,6 +430,102 @@ RZ_IPI void rz_core_clippy_print(RzCore *core, const char *msg) {
 	}
 }
 
+RZ_IPI void rz_help_calc_expr(void* data, const char* input) {
```

Comment:
I used this function directly inside the `rz_cmd_help` function in `"? "` case. Would it be a good idea to modify it's signature? I'll be like copying the code? 

Since I needed only the part which calculates expression and prints it to screen, I removed that case from `rz_cmd_help` and created this new function and then used this function there directly.